07-Customizing-AzSK-for-your-Org/Readme.md

```diff
@@ -460,7 +460,35 @@ image below:
 This change will be effective across your organization immediately. Anyone running AzSK commands (in fresh PS sessions)
 should see the new message. 
 
-##### b) Changing a control setting for specific controls 
+##### b) Customize output folder for my org
+Normally, when we run GSS for our org policy its security scan results goes to an folder 
+`%LOCALAPPDATA%\Microsoft\AzSKLogs\Sub_[yourSubscriptionName]E.g.`
+C:\Users\UserName\AppData\Local\Microsoft\AzSKLogs\Sub_[yourSubscriptionName]\20170331_142819. We can customize this location also so that our scan result log can be collected at a particular location given by you.
+
+###### Steps:
+
+i) Open the AzSk.json from your local org-policy folder. Path for example: D:\ContosoPolicies\Config\AzSK.json
+
+ii) Add new field OutputFolderPath and give an particular path where you want to store GSS/GRS security scan results 
+E.g.
+```PowerShell
+     "OutputFolderPath":  "Path"
+```
+iii) Save the file
+iv) Rerun the policy update or setup command (the same command you ran for the first-time setup)
+
+###### Testing: 
+
+Anyone in your org can now start a fresh PS console and again run IOP cmdlet to install updated org policy for your org and if he run GSS/GRS cmdlet then its security scan result log will be collected at a particular defined location in OutputFolderPath field added in azsk.json file.
```

Comment:
Why one would need to run IOP? Won't the policy will be effective to everyone in the org immediately?

Reply:
Done